Jenkinsfile

```diff
@@ -1,14 +1,13 @@
 #!groovy
 library "contrailWindows@$BRANCH_NAME"
 
-def ansibleExtraVars
-
 pipeline {
     agent none
 
     options {
         timeout time: 5, unit: 'HOURS'
         timestamps()
+        lock label: 'testenv_pool', quantity: 1
```

Comment:
Also wondering, whether there's a way to monitor how many locks are used. And if we want to monitor this.

Reply:
If there was an issue with testenv_pool we would see job queue growing.